src/utils/regex.js

```diff
@@ -1,22 +1,18 @@
 let _ = require('lodash/fp')
 
-let toSafeRegex = _.flow(
-  _.replace(/[.?+*|{}[\]()]/g, '\\$&'),
-  _.map(x => (x.match(/[A-Za-z]/) ? `[${_.toUpper(x)}${_.toLower(x)}]` : x)),
-  _.join('')
-)
+let toSafeRegex = _.flow(_.replace(/[.?+*|{}[\]()]/g, ''))
 
-let regexPartsForWords = _.flow(
-  _.replace(/\s\s+/g, ' '),
```

Comment:
We should probably still strip extra spaces

Reply:
⛳ : the split on `/s+/` has the same effect